pkg/ebpf/events/events.go

```diff
@@ -207,6 +209,18 @@ func capturePolicyEvents(ringbufferdata <-chan []byte, log logr.Logger, enableCl
 	}(ringbufferdata)
 }
 
+func logPolicyEventsWithK8sMetadata(log logr.Logger, message *string, nodeName, srcIP string, srcPort int, destIP string, destPort int, protocol, verdict string) {
+	srcName, srcNS, err := utils.GetPodMetadata(srcIP)
+	if err != nil {
+		log.Info("Failed to get source name and namespace metadata", "source IP:", srcIP)
```

Comment:
Let's do away with this. We're already making an assumption that an unknown IP is an external IP to the cluster...